test/accessibility.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { parameters } from '@playwright/test-runner/out/fixtures';
```

Comment:
do we have to import it from out?

Reply:
no, this is my autocomplete. lint found this as well.